web/app/components/workflow/header/index.tsx

```diff
@@ -58,13 +58,13 @@ const Header: FC = () => {
 
   const handleShowFeatures = useCallback(() => {
     const {
+      showFeaturesPanel,
       isRestoring,
       setShowFeaturesPanel,
     } = workflowStore.getState()
     if (getNodesReadOnly() && !isRestoring)
       return
-
-    setShowFeaturesPanel(true)
+    setShowFeaturesPanel(!showFeaturesPanel)
```

Comment:
The features panel can now be closed by clicking on it again.


https://github.com/langgenius/dify/assets/18554747/7e275461-25e4-4866-a77d-5ebdaee00609